libs/common-components/src/lib/zoom.directive.ts

```diff
@@ -34,17 +30,15 @@ export class ZoomDirective implements OnInit, OnDestroy, AfterViewInit {
   }
 
   private setStyles(zoom) {
-    if (! this.zoomActive) {
-      return;
```

Comment:
i think these lines are important and we shouldn't remove them

Reply:
Could you please clarify the mission of this check in the current implementation where i relying on scaling for Chrome/FF and on zoom for Edge?
I plan to merge this PR in the parent one where we can continue discussion.